### ISSUE TITLE
Support for AMQPS (SSL/TLS connections)

### ISSUE THREAD

**Issue by mikeatlas** [skybet/amqpea#10]
Do you have an example of connecting over `amqps://` scheme URI instead of plain `amqp://`?

**Comment by glenjamin** [comment]
I don't think this is currently possible - while bramqp supports this by being passed a secure socket - the current URI parsing / connection code doesn't.

I'd happily accept a PR for this - the TLS example on https://github.com/bakkerthehacker/bramqp/blob/master/doc/Setup.md#initialize-socket should help.
